e2e/scenario/filters1.spec.js

```diff
@@ -0,0 +1,1521 @@
+    }
+  });
+
+  it("step0v1: check name is Accessories - check with wrong dom properties", async function () {
+    await browser.sleep(5000);
+    var ui5ControlProperties = {
+      "elementProperties":{"metadata":"sap.m.StandardListItem","mProperties":{
+        "viewId": "cont*cart*", 
+        "title":[{"path":"CategoryName"}],
+        "bindingContextPath":"/ProductCategories*"},
+      "domProperties": {
+        "nodeName": "li",
+        "data-sap-ui": "*categoryList-0",
+        "role": "option",
+        "class": "*sapMLIB1*sapMSLI",
+        "id": "__item1-container-cart---homeView--categoryList-0"
+      }
+      }
+    };
+    try {
+      await common.locator.getDisplayedElement(ui5ControlProperties, 0, 5000);
+      await expect(true).toBe(false);
+    } catch (error) {
+      await expect(true).toBe(true);	
+    }
+  });
+ 
+  it("check getDisplayedChildElement 1", async function(){
+    var ui5ControlProperties = {
+      "elementProperties":{"metadata":"sap.m.List","mProperties":{"items":[{"path":"/ProductCategories"}]}},
+      "parentProperties":{"metadata":"sap.m.Page","mProperties":{"title":[{"path":"i18n>homeTitle"}]}}
+    };
+  
+    var ui5ControlProperties2 = {
+      "elementProperties":{"metadata":"sap.m.StandardListItem","mProperties":{"title":[{"path":"CategoryName"}],"bindingContextPath":"/ProductCategories*"}},
+    };
+	
+    const nameField = await common.locator.getDisplayedChildElement(ui5ControlProperties, ui5ControlProperties2);
+    var attribute = "title";   //eg: title, text, value etc.
+    var compareValue ="Accessories";   //expected value
+    await expect(await nameField.getAttribute("data-" + attribute)).toBe(compareValue);
+  
+  });
+
+  it("step0v1: check name is Accessories - use UI5 Control properties", async function () {
+    var ui5ControlProperties = {
+      "elementProperties":{"metadata":"sap.m.StandardListItem","mProperties":{
+        "viewId": "container-cart---app", 
+        "title":[{"path":"CategoryName"}],
+        "bindingContextPath":"/ProductCategories*"}
+      }
+    };
+    var elem = await common.locator.getDisplayedElement(ui5ControlProperties);
+    var attribute = "title";
+    var compareValue ="Accessories";
+    var val = await ui5ControlConverter.getControlProperty(elem, attribute);
+    await expect(val).toBe(compareValue);
+  });
+
+  it("step0v1: check name is Accessories - use UI5 Aggregation Control properties", async function () {
+    var ui5ControlProperties = {
+      "elementProperties":{"metadata":"sap.m.StandardListItem","mProperties":{
+        "viewId": "container-cart---app", 
+        "title":[{"path":"CategoryName"}],
+        "bindingContextPath":"/ProductCategories*"}
+      }
+    };
+    var elem = await common.locator.getDisplayedElement(ui5ControlProperties);
+    var attribute = "tooltip";
+    var compareValue ="Open category Accessories";
+    var val = await ui5ControlConverter.getControlAggregationProperty(elem, attribute);
```

Comment:
we have a getProperty method in uiveri5/control.js, perhaps we should just extend it with getters for aggregations and bindings?

Reply:
yes thats another point that in my list